transport-native-epoll/src/main/java/io/netty/channel/epoll/AIOContext.java

```diff
@@ -31,12 +33,18 @@
 
     private final long address;
     private long nextId;
+    private long outstanding;
```

Comment:
Do we need this counter, isn't the size of `outstandingRequests` sufficient, given that the code executes in the same thread?

Reply:
Just more efficient to check a counter vs map.size()

Reply:
True in the general case, but `LongObjectHashMap` keeps the size cached, so I don't think this would have a performance impact, although I agree this is a super-hot code path.